playground/src/main/java/mx/yellowme/youst/playground/ui/chart/utils/ChartBuilder.kt

```diff
@@ -15,11 +16,11 @@ import mx.yellowme.youst.playground.domain.ChartType
  */
 object ChartBuilder {
 
-    fun buildChart(activity: AppCompatActivity): View? {
+    fun buildChart(activity: AppCompatActivity): BarLineChartBase<*>? {
         with(activity) {
             parseSettingsJson(this).run {
                 instanceTypeChart(this, activity).also {
```

Comment:
Si no me equivoco el return podría ir aquí 🤔 Para aprovechar que el also retorna una instancia del objeto.

Reply:
Me parece que ya tiene un `it` más abajo... yo cuestionaría si es necesario "nestear" los scopes, ¿Qué opinan?

Reply:
Hasta se podría hacer esto, que opinan? 
```
    fun buildChart(activity: FragmentActivity): BarLineChartBase<*>? =
        parseSettingsJson(activity).run {
            setConfiguration(this, instanceTypeChart(this, activity), activity)
        }
```

o esto 

```
    fun buildChart(activity: FragmentActivity): BarLineChartBase<*>? {
        return parseSettingsJson(activity).run {
            setConfiguration(this, instanceTypeChart(this, activity), activity)
        }
    }
```

Reply:
Me hace mucho ruido que `activity` sea enviado dos veces como parámetro